sky/backends/backend_utils.py

```diff
@@ -1323,8 +1324,8 @@ def generate_cluster_name():
     return f'sky-{uuid.uuid4().hex[:4]}-{get_cleaned_username()}'
 
 
-def get_cleaned_username() -> str:
-    """Cleans the current username to be used as part of a cluster name.
+def get_cleaned_username(username: str = "") -> str:
```

Comment:
```suggestion
def get_cleaned_username(username: str = '') -> str:
```
stylenit: we use single quotes throughout